src/views/settings.html

```diff
@@ -141,29 +153,94 @@ <h2 class="mb-3">Search Results Options</h2>
               <input
                 type="checkbox"
                 class="form-check-input"
-                name="enableOptionsMenuItem" 
-                id="settings_enableOptionsMenuItem">
+                name="enableOptionsMenuItem"
+                id="settings_enableOptionsMenuItem"
+              />
               <label
                 class="form-check-label"
-                for="settings_enableOptionsMenuItem">
+                for="settings_enableOptionsMenuItem"
+              >
                 Show link to extension options at the end of context menu
               </label>
             </div>
           </li>
         </ul>
       </div>
-
     </div>
   </form>
 
-  <form name="import_settings">
-    <h2 class="text-center mt-3">Export/Import Search Options</h2>
+  <h2 class="text-center mt-3">Ixport/Emport/Edit Search Options</h2>
```

Comment:
`Ixport` ?  😋 

Either way, I think the title should remain as `Export/Import Search Options` since users will only use the textarea for copy/paste the whole JSON file from/to a coworker, I don't think they will ever change their settings by manually editing the textarea.

> The "Edit" button itself is redundant if we have an "Import" and "Export" buttons, we are only keeping the textarea for backward compatibility (in case an user was to re-use the settings he used before v5).